sdk/spring/azure-spring-integration-eventhubs/src/test/java/com/azure/spring/integration/eventhub/converter/EventHubMessageConverterTest.java

```diff
@@ -5,31 +5,38 @@
 
 import com.azure.messaging.eventhubs.EventData;
 import com.azure.spring.integration.core.AzureHeaders;
+import com.azure.spring.integration.core.EventHubHeaders;
 import com.azure.spring.integration.core.converter.AzureMessageConverter;
 import com.azure.spring.integration.test.support.AzureMessageConverterTest;
 import org.junit.Test;
 import org.springframework.messaging.MessageHeaders;
 import org.springframework.messaging.support.NativeMessageHeaderAccessor;
 import org.springframework.util.LinkedMultiValueMap;
 
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class EventHubMessageConverterTest extends AzureMessageConverterTest<EventData> {
 
     private static final String EVENT_DATA = "event-hub-test-string";
-    private static final int PARTITION_ID = 1;
+    private static final String RAW_PARTITION_ID = "1";
     private static final String NATIVE_HEADERS_SPAN_ID_KEY = "spanId";
     private static final List<String> NATIVE_HEADERS_SPAN_ID_VALUE = Arrays.asList("spanId-1", "spanId-2");
     private static final String NATIVE_HEADERS_SPAN_TRACE_ID_KEY = "spanTraceId";
     private static final List<String> NATIVE_HEADERS_SPAN_TRACE_ID_VALUE = Arrays
             .asList("spanTraceId-1", "spanTraceId-2");
-
+    private static final Instant ENQUEUED_TIME = Instant.now().minus(1, ChronoUnit.DAYS);
+    private static final Long OFFSET = 1234567890L;
+    private static final Long SEQUENCE_NUMBER = 1234567890L;
```

Comment:
nit: SEQUENCE should be smaller than OFFSET, seq number indicates the message is #n in the queue, but offset is calculated using bytes.